ts/store/reducers/content.ts

```diff
@@ -232,11 +232,17 @@ export default function content(
       };
 
     case getType(contentMunicipalityLoad.request):
+      const codiceCatastale = state.municipality.codiceCatastale;
+      const municipalityData = state.municipality.data;
       return {
         ...state,
         municipality: {
```

Comment:
```suggestion
        municipality: {
          codiceCatastale: pot.toLoading(codiceCatastale),
          data: pot.toLoading(municipalityData)
        }
```
we need to preserve the _loading_ state
In that way the `pot` could be loading (noneLoading, if there is no value) and previous value (some, error, loading) if there is a value

Reply:
Got it! 

Just for knowledge my code can bring to some wrong behaviour?

Reply:
Yes it does. It doesn't represent correctly the request status.
Only when the store is empty (`isNone`) the status will be right (`noneLoading`)
In all other cases (`some`, `error`) when a request start we won't have the status loading.

This could be a problem if we have to show a loader in the meantime: it will be shown only under specific scenario